pkg/reconciler/dependencybuild/scripts/deploy.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```

Comment:
so I'm guessing that the `Move deployment out of the cache to an task` highlight in this PR's description includes this deletion, along with the addition of these tar's to the specific gradle / mavne scripts below

is that correct @stuartwdouglas ?

Reply:
Yes, this is now a step in the golang code that calls the 'build-request-processor' with specific args to do the deployment.